includes/theme-compat/header.php

```diff
@@ -0,0 +1,52 @@
+<?php
+//require_once (dirname(__FILE__).'/config/config.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/config.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/functions.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/db-connect.php');
```

Comment:
You actually might want to require the db-connect.php file before the functions.php file
```suggestion
require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/db-connect.php');
require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/functions.php');
```

Reply:
I changed the calling position.